engine/audio/src/audiocapture.cpp

```diff
@@ -166,22 +168,21 @@ double AudioCapture::fillBandsData(int number)
             magnitudeSum += qSqrt((((fftw_complex*)m_fftOutputBuffer)[i][0] * ((fftw_complex*)m_fftOutputBuffer)[i][0]) +
                                   (((fftw_complex*)m_fftOutputBuffer)[i][1] * ((fftw_complex*)m_fftOutputBuffer)[i][1]));
         }
-        double bandMagnitude = (magnitudeSum / subBandWidth);
+        bandMagnitude = (magnitudeSum / subBandWidth);
         m_fftMagnitudeMap[number].m_fftMagnitudeBuffer[b] = bandMagnitude;
-        if (maxMagnitude < bandMagnitude)
-            maxMagnitude = bandMagnitude;
```

Comment:
I removed the maxMagnitude computation so we can compute it later and keep the result of the current band instead in the return value.